Makefile.am

```diff
@@ -87,11 +88,7 @@ ucd_data_fetch_CFLAGS = $(AM_CFLAGS)
 SYSTEMD_DIR=$(prefix)/lib/systemd/system/
 systemdsystemunitdir = @SYSTEMD_SYSTEMUNITDIR@
 systemdsystemunit_DATA = data/ucd.service \
-	data/ucd-aws.service \
-	data/ucd-oci.service \
-	data/ucd-tencent.service \
-	data/ucd-equinix.service \
-	data/ucd-aliyun.service
+			 data/ucd@.service
```

Comment:
Yes but you do need to create this file. It's basically the same file but `%I` instead of where `oci` etc. was in the original.

Reply:
Whoops, I did create `data/ucd@.service.in` but forgot to `git add`.

Reply:
I guess `%I` is slightly preferable to `%i`?

Reply:
Yes. If there's ever a name that needs escaping, `%I` would make it workable.